zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -903,6 +903,16 @@ public int getSnapRetainCount() {
         return snapRetainCount;
     }
 
+    /**
+     * Use {@link #getPurgeIntervalInMs()} instead.
+     *
+     * @return purge interval in hour unit or 0 if less than one hour.
+     */
+    @Deprecated
+    public int getPurgeInterval() {
```

Comment:
This is the deleted version. https://github.com/apache/zookeeper/pull/2178/files